ldelf/ta_elf.c

```diff
@@ -24,6 +25,21 @@
 #include "ta_elf.h"
 #include "unwind.h"
 
+/*
+ * Layout of a 32-bit struct dl_phdr_info for a 64-bit ldelf to access a 32-bit
+ * TA
+ */
+struct _dl_phdr_info32 {
```

Comment:
Why the leading `_`?

Reply:
No good reason really... This is an internal structure while `dl_phdr_info` is an exported type available to TAs via `<link.h>`.  I felt I had to tag it somehow but it's not useful in any way in fact ;-)